Multiple defs

higs4281 opened the pull request:
Adds support for multiple definitions and stops the brain load cycling.
Addresses #6  and #7

higs4281 committed:
ignore acro.priv.json

higs4281 committed:
adds ability to add and delete alternate definitions

higs4281 committed:
explain definition deletion if there is only one definition

higs4281 commented:
@contolini @hillaryj @djdrisco what say you?

contolini commented on src/acrogov.coffee:
```diff
@@ -39,18 +42,35 @@ class AcroBot
         @loadPublicAcronyms()
         console.log("loaded public acronyms")
```
Change all the `console.log()` statements to `@robot.logger.debug()` to reduce console noise. They'll be printed when `HUBOT_LOG_LEVEL="debug"`.

higs4281 committed:
removed console.log statements

I don't think they remain useful. In debug mode they're lost
in a sea of messages, and they make our extensive test suite fail.
I say lose 'em.

contolini committed:
Revamp all the tests

contolini committed:
Merge branch 'multiple-defs' of github.com:cfpb/hubot-acrogov into multiple-defs

higs4281 commented:
@contolini gone

contolini commented:
:+1:

I just added a bunch of tests. I'll push 'em up shortly.

contolini committed:
Add npm and Travis CI badges to readme

contolini committed:
Update Travis' Node versions

contolini commented:
@higs4281 pull down master to ensure you're up-to-date and run `npm version [major|minor|patch]` (choose one, your call) followed by `npm publish` to publish these changes to npm.

Then in chat do `cfpbot update`, wait a couple minutes for it to update its npm dependencies, and then `cfpbot die`.

Nice work with this update btw!

higs4281 commented:
@contolini those tests are great. i should not be rewarded for being facetious.

contolini commented:
@higs4281 Don't flatter yourself! I was 90% done with the tests before I read your facetiousness. I figured out the good Hubot test structure while building https://github.com/catops/generator-catops-script.
